miden/src/cli/debug/mod.rs

```diff
@@ -56,18 +56,28 @@ impl DebugCmd {
         let mut rl =
             Editor::<()>::with_config(rl_config).expect("Readline couldn't be initialized");
 
+        println!("Welcome! Enter `h` for help.");
+
         loop {
             match rl.readline(">> ") {
                 Ok(command) => match DebugCommand::parse(&command) {
-                    Ok(command) => {
+                    Ok(Some(command)) => {
                         if !debug_executor.execute(command) {
                             println!("Debugging complete");
                             break;
                         }
                     }
-                    Err(err) => println!("{err}"),
+                    Ok(None) => (),
+                    Err(err) => eprintln!("{err}"),
                 },
-                Err(err) => println!("malformed command - failed to read user input: {}", err),
+                Err(ReadlineError::Interrupted) => {
+                    //eprintln!("CTRL-C");
```

Comment:
What is the expected behaviour here? Did you intend on adding a print or a break?

Reply:
Better replace this by a comment. The idea is to do nothing when the user inputs ctrl+c, and halt whatever he is inputting.